app/src/main/java/com/google/android/stardroid/StardroidApplication.java

```diff
@@ -46,51 +52,84 @@
  * @author John Taylor
  */
 public class StardroidApplication extends Application {
-  private static final String TAG = MiscUtil.getTag(StardroidApplication.class);
-  private static final String PREVIOUS_APP_VERSION_PREF = "previous_app_version";
-  private static final String NONE = "Clean install";
-  private static final String UNKNOWN = "Unknown previous version";
-
-  @Inject SharedPreferences preferences;
-  // We keep a reference to this just to start it initializing.
-  @Inject LayerManager layerManager;
-  @Inject Analytics analytics;
-  @Inject SensorManager sensorManager;
-
-  // We need to maintain references to this object to keep it from
-  // getting gc'd.
-  @Inject PreferenceChangeAnalyticsTracker preferenceChangeAnalyticsTracker;
-  private ApplicationComponent component;
-
-  @Override
-  public void onCreate() {
-    Log.d(TAG, "StardroidApplication: onCreate");
-    super.onCreate();
-
-    component = DaggerApplicationComponent.builder()
-        .applicationModule(new ApplicationModule(this))
-        .build();
-    component.inject(this);
-
-    Log.i(TAG, "OS Version: " + android.os.Build.VERSION.RELEASE
-            + "(" + android.os.Build.VERSION.SDK_INT + ")");
-    String versionName = getVersionName();
-    Log.i(TAG, "Sky Map version " + versionName + " build " + getVersion());
-
-    // This populates the default values from the preferences XML file. See
-    // {@link DefaultValues} for more details.
-    PreferenceManager.setDefaultValues(this, R.xml.preference_screen, false);
-
-    setUpAnalytics(versionName);
-
-    performFeatureCheck();
+    private static final String TAG = MiscUtil.getTag(StardroidApplication.class);
```

Comment:
Sorry - looks like you've done a big reformatting here. Please don't do that - it makes the change history harder to understand and can make merges more difficult.  Doing a one off refactor to reformat is sometimes OK if a file is badly formatted, but it's best not to mix it up with functionality changes.

Reply:
@jaydeetay i have only reformatted the code using ctrl+alt+L. Nothing much.

Reply:
@jaydeetay I understood what you were talking about. Changes made :+1: